src/roaring64.c

```diff
@@ -803,6 +803,69 @@ bool roaring64_bitmap_run_optimize(roaring64_bitmap_t *r) {
     return has_run_container;
 }
 
+typedef struct min_max_sum64_s {
+    uint64_t min;
+    uint64_t max;
+    uint64_t sum;
+} min_max_sum64_s;
+
+static bool min_max_sum64_fnc(uint64_t value, void *param) {
+    min_max_sum64_s *mms = (min_max_sum64_s *)param;
+    if (value > mms->max) mms->max = value;
+    if (value < mms->min) mms->min = value;
```

Comment:
Roaring64 bitmaps are iterated in ascending order, the min value will just be the first value, and the max will be the last.

Hmm. I see this matches the implementation of the 32 bit statistics function..

Reply:
maybe it is worth to open an issue to fix the roaring32 statistics perf will help fix it when we finish with this

Reply:
Referenced in #619